lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -217,13 +218,19 @@ IncrementalJIT::getSymbolAddressWithoutMangling(llvm::StringRef Name,
   if (auto Sym = getInjectedSymbols(Name))
     return Sym;
 
+  const std::string SymName = Name;
```

Comment:
What about making `DLSym` taking a `llvm::StringRef`, this would factor out the need of `CStr` variable, IIUC. In general we are very much in favor of using `llvm::StringRef`-s where possible.